updates/1/updates.sql

```diff
@@ -1 +1,28 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 1604022038, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", "C", "Alchemist", "MIST", 1612643126, NULL, "alchemist", "mist", "0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB");
+*
+UPDATE assets SET swapped_for="_ceth_0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0" WHERE identifier="_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5";/*Introduce TRB token swap*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5", "C", "Tellor", "TRB", 1564671864, NULL, "tellor", "trb", "0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5");
```

Comment:
When you add the full insertion for an already existing insertion please take a look at the currently existing DB entry. The cryptocompare entry is NULL for Tellor:

```
    "TRB": {
        "coingecko": "tellor",
        "ethereum_address": "0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5",
        "ethereum_token_decimals": 18,
        "name": "Tellor Tributes",
        "started": 1564671864,
        "symbol": "TRB",
        "type": "ethereum token"
    },
```

That means that the symbol is attempted by default. (If it's `""` then that means completely disabled -- don't query). This is a bit annoying and we should improve it in the future. But it's documented also in the api docs I believe.

Should be fine to leave it like that as it's more complete of an entry and this will only be touched if the user deleted Tellor from their DB but I am just letting you know of this difference.